avocado/core/runners/requirement_package.py

```diff
@@ -105,17 +106,15 @@ def _run_software_manager(self, cmd, package, queue):
         queue.put(output)
 
     def run(self):
-        yield self.prepare_status('started')
+        yield messages.get_started_message()
```

Comment:
Following my proposal this would become something like `yield StartedMessage.get()` or just `yield StartedMessage()`